### ISSUE TITLE
Make todo lists accessible

### ISSUE THREAD

**Issue by mmotyczynska** [ckeditor/ckeditor5#12395]
## 📝 Provide a description of the improvement

The accessibility of todo lists has to be improved.

For example, it is not obvious what keyboard shortcut to use to select the checkbox. The user should be able to find out that the shortcut is `Ctrl+Enter`.

## 📃 Other details

* Browser: …
* OS: …
* CKEditor version: …
* Installed CKEditor plugins: …

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.
